paasta_tools/frameworks/native_scheduler.py

```diff
@@ -545,25 +534,27 @@ def create_driver(
     system_paasta_config,
     implicit_acks=False
 ):
-    framework = mesos_pb2.FrameworkInfo()
-    framework.user = ""  # Have Mesos fill in the current user.
-    framework.name = framework_name
-    framework.failover_timeout = 604800
-    framework.id.value = find_existing_id_if_exists_or_gen_new(framework.name)
-    framework.checkpoint = True
-
-    credential = mesos_pb2.Credential()
-    credential.principal = system_paasta_config.get_paasta_native_config()['principal']
-    credential.secret = system_paasta_config.get_paasta_native_config()['secret']
+    master_uri = '{}:{}'.format(
+        mesos_tools.get_mesos_leader(), mesos_tools.MESOS_MASTER_PORT
+    )
 
-    framework.principal = system_paasta_config.get_paasta_native_config()['principal']
+    framework = Dict(
+        user=getpass.getuser(),
```

Comment:
this looks different. should it not be `None`? I don't think it is supposed to literally be the current user.